curvefs/conf/client.conf

```diff
@@ -144,14 +144,19 @@ diskCache.maxUsableSpaceBytes=107374182400
 diskCache.cmdTimeoutSec=300
 # directory of disk cache
 diskCache.cacheDir=/mnt/curvefs_cache  # __CURVEADM_TEMPLATE__ /curvefs/client/data/cache __CURVEADM_TEMPLATE__  __ANSIBLE_TEMPLATE__ /mnt/curvefs_disk_cache/{{ 99999999 | random | to_uuid | upper }} __ANSIBLE_TEMPLATE__
-# the write bps of disk cache
-diskCache.maxFlushBytes=83886080
-# the write iops of disk cache
-diskCache.maxFlushIops=100000000
-# the read bps of disk cache
-diskCache.maxReadFileBytes=83886080
-# the read iops of disk cache
-diskCache.maxReadFileIops=100000000
+
+# the write throttle bps of disk cache
+diskCache.avgFlushBytes=83886080
```

Comment:
mark as  
【the write throttle bps of disk cache, default 80MB/s】
to more readable

Reply:
fix